src/main/java/roomescape/reservation/presentation/dto/ReservationResponse.java

```diff
@@ -1,11 +1,12 @@
 package roomescape.reservation.presentation.dto;
 
 import java.time.LocalDate;
+import roomescape.member.presentation.dto.MemberNameResponse;
 import roomescape.reservation.domain.Reservation;
 
 public class ReservationResponse {
     private Long id;
-    private String name;
+    private MemberNameResponse member;
```

Comment:
```suggestion
    private MemberNameResponse memberName;
```
이렇게 되는게 좀 덜헷갈릴 것 같아요!